exp-timing/femto.py

```diff
@@ -39,17 +39,16 @@ def __init__(self, nx='NULL'):
         secondary_calibration_c = dict() # Cosine term for calibration
         use_drift_correction = dict() # Allows drift correction feature to be turned on/off individually for each locker. 
         drift_correction_signal = dict() # Drift correction value that is pulled from the time_tool.py script. 
-        drift_correction_multiplier = dict() # Multiplier for the drift correction value. 
         drift_correction_value = dict() # Drift correction value in ns that is sent to a notepad PV.
-        drift_correction_offset = dict() # Offset factor for the drift correction value. 
-        drift_correction_gain = dict()  # Gain factor for drift correction value. 
-        drift_correction_dir = dict()  # Direction of correction, configurable based on the set-up of the timetool stage in a particular hutch. 
+        drift_correction_offset = dict() # Allows a fixed offset to be applied to the 'fs' value from the timetool script.
+        drift_correction_gain = dict() # 0 is disable.
```

Comment:
This is not clear. Maybe you mean it's a bool value?

Reply:
Updated to specify that 0 disables drift correction feedback entirely